src/style.css

```diff
@@ -1 +1,68 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  border-color: #fff;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: space-between;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  list-style: none;
+  margin-right: 20px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  line-height: 60px;
+  display: flex;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: black;
+  transition: color 0.4s;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.icon {
+  margin: 10px 0 10px 50px;
+  width: 40px;
+  height: 40px;
+}
+
+.is-active {
+  position: relative;
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  bottom: 0;
+  width: 100%;
+  position: absolute;
+  height: 4px;
+  background-color: #00acdc;
+  border-radius: 8px;
```

Comment:
Pay attention to your CSS properties order
https://9elements.com/css-rule-order/